tests/operators/test_snowflake_append.py

```diff
@@ -103,30 +123,38 @@ def wait_for_task_finish(self, dr, task_id):
 
     def run_append_func(self, columns, casted_columns):
         cwd = pathlib.Path(__file__).parent
-
         with self.dag:
+            self.TABLE_1 = Table(
+                table_name=self.TABLE_1_NAME,
+                database=os.getenv("SNOWFLAKE_DATABASE"),
+                schema=os.getenv("SNOWFLAKE_SCHEMA"),
+                warehouse=os.getenv("SNOWFLAKE_WAREHOUSE"),
+                conn_id="snowflake_conn",
+            )
+
+            self.TABLE_2 = Table(
+                table_name=self.TABLE_2_NAME,
+                database=os.getenv("SNOWFLAKE_DATABASE"),
+                schema=os.getenv("SNOWFLAKE_SCHEMA"),
+                warehouse=os.getenv("SNOWFLAKE_WAREHOUSE"),
+                conn_id="snowflake_conn",
+            )
+
             load_main = aql.load_file(
                 path=str(cwd) + "/../data/homes_main.csv",
-                output_table=Table(
-                    table_name="test_append_1",
-                    conn_id="snowflake_conn",
-                    schema="tmp_astro",
-                ),
+                file_conn_id="",
+                output_table=self.TABLE_1,
             )
             load_append = aql.load_file(
```

Comment:
Can you do something similar hear to what you did in merge where you load a raw table into snowflake once and then just copy that table at test-time? I think that can save us some time on CI/CD as snowflake loads are comparatively slow.

Reply:
@dimberman, I have made the required changes.